### PR TITLE
Fix CutAngle class Javadoc

### PR DESCRIPTION
**Pull request by ivanshuba**
This commit modifies the Javadoc such that:

1. Ordered list is rendered correctly
2. Special symbols such as \pi, \implies are rendered correctly

Note: Requires checking by the author of the original class to make sure the description explaining Point classification contains correct math symbols in all places.

### CODE REVIEW

**Comment by aherbert** [comment]
Note that all javadoc within the `\( \)` parentheses are intentional. These are formatting tags for [MathJax](https://www.mathjax.org/). This can make the document less readable where the rendering is not performed. When rendered with MathJax within the HTML generated javadoc then the document is as intended.

This update was made after the last official release and so there are no corresponding javadoc pages on the geometry site. You can view the documents locally (assuming you have the build tools of Maven and a JDK) using:

```
cd commons-geometry-spherical/
mvn javadoc:javadoc
open target/site/apidocs/org/apache/commons/geometry/spherical/oned/CutAngle.html
```

**Comment by ivanshuba** [comment]
@aherbert Thank you for the quick response and the explanation! I've followed your guideline and were able to render documentation with correctly represented symbols successfully. 

The description now makes sense to me. Before this, I thought the description used $\pi$, so the following statements confused me when I reconstructed symbols in the description myself: 
> ...(1) define a second, implicit cut point at $\pi$
> ...the hyperplane at $\pi$ is unique
> ...the test point, or both lie very close to $\pi$
> ...points are considered equivalent to $\pi$ if their normalized azimuth are close to either $\pi$ or $2\pi$

Now, everything fits.
> ...(1) define a second, implicit cut point at $0\pi$
> ...the hyperplane at $0\pi$ is unique
> ...the test point, or both lie very close to $0\pi$
> ...points are considered equivalent to $0\pi$ if their normalized azimuth are close to either $0\pi$ or $2\pi$
